workshop_sections/mnist_series/mnist_simple.py

```diff
@@ -53,11 +53,11 @@ def main(_):
     # So here we use tf.nn.softmax_cross_entropy_with_logits on the raw
     # outputs of 'y', and then average across the batch.
     cross_entropy = tf.reduce_mean(
-        tf.nn.softmax_cross_entropy_with_logits(y, y_))
+        tf.nn.softmax_cross_entropy_with_logits(logits=y, labels=y_))
```

Comment:
As they're "named args" I don't think it matters? I suspect that's why they made that change, as people were probably forgetting which is which.